.github/workflows/terraform-module-continuous-integration.yaml

```diff
@@ -36,10 +36,10 @@ jobs:
         run: tflint --init && tflint -f compact
         env:
           GITHUB_TOKEN: ${{ github.token }}
-      - name: Check Terraform Security (TFsec)
-        uses: aquasecurity/tfsec-action@v1.0.3
-      - name: Check Terraform Security (Checkov)
-        uses: bridgecrewio/checkov-action@v12
-        with:
-          directory: .
-          framework: terraform
+      # - name: Check Terraform Security (TFsec)
```

Comment:
We can remove this, if we ever need it we can check the git history but it will most likely need a relook because of new versions etc.

Reply:
Sounds good, removing it and having it as a backlog